src/test/unit/test.js

```diff
@@ -1091,9 +1091,48 @@ describe('Client', function() {
       })
     })
 
```

Comment:
can you write the test cases in functional tests? we are moving away from unit tests to functional tests for API testing

`minio-js/src/test/functional/functional-tests.js`